src/components/sidebar/CallTreeSidebar.js

```diff
@@ -432,6 +452,7 @@ class CallTreeSidebarImpl extends React.PureComponent<Props> {
                 </div>
               </h4>
               <CategoryBreakdown
+                isTotal={true}
```

Comment:
Instead of passing a boolean, you can pass a string, that would be the key of the breakdown to fetch and change.
The key could be used both in the action to save the information, and when retrieving the information from the selector.